src/test/groovy/edu/ie3/datamodel/io/processor/input/AssetInputProcessorTest.groovy

```diff
@@ -0,0 +1,286 @@
+package edu.ie3.datamodel.io.processor.input
+
+import edu.ie3.datamodel.models.StandardUnits
+import edu.ie3.datamodel.models.input.NodeInput
+import edu.ie3.datamodel.models.input.connector.LineInput
+import edu.ie3.datamodel.models.input.connector.SwitchInput
+import edu.ie3.datamodel.models.input.connector.Transformer2WInput
+import edu.ie3.datamodel.models.input.connector.Transformer3WInput
+import edu.ie3.datamodel.models.input.system.BmInput
+import edu.ie3.datamodel.models.input.system.ChpInput
+import edu.ie3.datamodel.models.input.system.EvInput
+import edu.ie3.datamodel.models.input.system.FixedFeedInInput
+import edu.ie3.datamodel.models.input.system.HpInput
+import edu.ie3.datamodel.models.input.system.LoadInput
+import edu.ie3.datamodel.models.input.system.PvInput
+import edu.ie3.datamodel.models.input.system.StorageInput
+import edu.ie3.datamodel.models.input.system.WecInput
+import edu.ie3.test.common.GridTestData
+import edu.ie3.test.common.SystemParticipantTestData
+import edu.ie3.util.TimeTools
+import spock.lang.Specification
+
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+/**
+ * //ToDo: Class Description
+ *
+ * @version 0.1* @since 24.03.20
+ */
+class AssetInputProcessorTest extends Specification {
+
+
+    def "A ResultEntityProcessor should de-serialize a provided NodeInput correctly"() {
+        given:
+        TimeTools.initialize(ZoneId.of("UTC"), Locale.GERMANY, "yyyy-MM-dd HH:mm:ss")
+        def sysPartResProcessor = new AssetInputProcessor(NodeInput)
```

Comment:
Too much of copy, paste. ;-) Please go through the test and adapt the other places as well.

```suggestion
        def assetInputProcessor = new AssetInputProcessor(NodeInput)
```